fasttrips/FastTrips.py

```diff
@@ -14,6 +14,8 @@
 """
 import os, sys
 
+from functools import partial
+import pandas
```

Comment:
Same here....not sure we need.  Also, if we do import, we should `import pandas as pd` per tradition, me thinks.